projects/1.18/assets/ars-elemental/ars_elemental/lang/zh_cn.json

```
@@ -0,0 +1,147 @@
+{
+  "itemGroup.ars_elemental": "元素魔艺",
+  "item.ars_elemental.spell_horn": "巫师号角",
+  "item.ars_elemental.fire_focus": "火之核心",
+  "item.ars_elemental.water_focus": "水之核心",
+  "item.ars_elemental.air_focus": "气之核心",
+  "item.ars_elemental.earth_focus": "土之核心",
+  "item.ars_elemental.lesser_fire_focus": "次级火之核心",
+  "item.ars_elemental.lesser_water_focus": "次级水之核心",
+  "item.ars_elemental.lesser_air_focus": "次级气之核心",
+  "item.ars_elemental.lesser_earth_focus": "次级土之核心",
+  "item.ars_elemental.necrotic_focus": "腐坏核心",

```

```suggestion
  "item.ars_elemental.necrotic_focus": "通灵核心",
```
“necromancy”是“通灵”而非“腐坏”的意思

---

这里本意是该核心的外观为黑色与红色的颜色组合，并且该核心的效果所“通灵”召唤的生物都属于邪恶生物，个人认为“腐坏”能更好反应使用法师的腐败灵魂